pandas/_libs/tslibs/np_datetime.pxd

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+
+from numpy cimport int64_t, int32_t
+
+
+cdef extern from "../src/datetime/np_datetime.h":
+    ctypedef struct pandas_datetimestruct:
+        int64_t year
+        int32_t month, day, hour, min, sec, us, ps, as
+
+
+cdef check_dts_bounds(pandas_datetimestruct *dts)
```

Comment:
you need a void here?

Reply:
ok, saw your note, pls add to the list to add this as void (if possible / needed), not sure.